packages/hardhat-core/src/internal/core/config/config-loading.ts

```diff
@@ -68,6 +68,8 @@ export function loadConfigAndTasks(
     showSolidityConfigWarnings: false,
   }
 ): { resolvedConfig: HardhatConfig; userConfig: HardhatUserConfig } {
+  const { validateConfig, validateResolvedConfig } =
+    require("./config-validation") as ConfigValidationT;
```

Comment:
You can also do this and drop the top-level type import:

```ts
  const { validateConfig, validateResolvedConfig } =
    require("./config-validation") as typeof import("./config-validation");
```

I don't know why the same trick doesn't work for `ordinal` though.

(I don't have a strong preference, use whichever form you like.)

Reply:
Ah, doing this also means that you don't need to define and export that `ConfigValidationT` interface, so yeah, let's do this change.

Reply:
I'm not a fan of that syntax but 👍